tests/qchem/test_hamiltonians.py

```diff
@@ -17,26 +17,46 @@
 # pylint: disable=too-many-arguments,too-few-public-methods
 import pytest
 
+import numpy as np
+
 import pennylane as qml
 from pennylane import I, X, Y, Z
-from pennylane import numpy as np
+from pennylane import numpy as pnp
 from pennylane import qchem
 from pennylane.fermi import from_string
 
 
+def test_molecular_hamiltonian_numpy_data():
+    """Test that if numpy data is used with molecular hamiltonian, that numpy data is outputted."""
+    symbols = ["H", "H"]
+    coordinates = np.array([[0.0, 0.0, -0.6614], [0.0, 0.0, 0.6614]])
```

Comment:
Not blocking, but to be safe, we might use parametrize and test both coordinate formats:

```
@pytest.mark.parametrize(
    "coordinates", [np.array([[0.0, 0.0, -0.6614], [0.0, 0.0, 0.6614]]), np.array([0.0, 0.0, -0.6614, 0.0, 0.0, 0.6614])]
```

Reply:
updated.